README.md

```diff
@@ -12,7 +12,7 @@ DKPro Argumentation is available from Maven Central. For working with the UIMA t
 <dependency>
     <groupId>de.tudarmstadt.ukp.dkpro.argumentation.types</groupId>
     <artifactId>de.tudarmstadt.ukp.dkpro.argumentation.types</artifactId>
-    <version>0.0.2</version>
+    <version>0.0.3-SNAPSHOT</version>
```

Comment:
Is is possible to "re-publish" a version on Maven Central?-- I thought this wasn't possible/allowed.